src/analysis/function_parameters.rs

```diff
@@ -242,10 +249,16 @@ pub fn analyze(
             let array_name = nameutil::mangle_keywords(&array_name[..]);
             add_rust_parameter = false;
 
+            let nullable = function_parameters
+                .iter()
+                .find(|p| p.name == array_name)
+                .unwrap()
```

Comment:
This panics when running `./generator.py --gir-path path/to/gir` while generating code for `glib`:

```
[...]
[WARN  libgir::analysis::functions] `g_variant_new_from_data`: destructor without linked callback
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', src/analysis/function_parameters.rs:255:18
stack backtrace:
   0: rust_begin_unwind
             at /rustc/88f19c6dab716c6281af7602e30f413e809c5974/library/std/src/panicking.rs:493:5
   1: core::panicking::panic_fmt
             at /rustc/88f19c6dab716c6281af7602e30f413e809c5974/library/core/src/panicking.rs:92:14
   2: core::panicking::panic
             at /rustc/88f19c6dab716c6281af7602e30f413e809c5974/library/core/src/panicking.rs:50:5
   3: libgir::analysis::function_parameters::analyze
   4: libgir::analysis::functions::analyze_function
   5: libgir::analysis::functions::analyze
   6: libgir::analysis::run
   7: gir::main
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
```

Reply:
Line 239 to 247 try to get the array name in various ways. It would probably be best to directly return the `nullable` in those places.